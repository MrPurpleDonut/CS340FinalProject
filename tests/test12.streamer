text file1data << args[0];
text file2data << args[1];
if (file1data == file2data)
{
  print "A match!";
}
else
{
  print "Different!";
}
