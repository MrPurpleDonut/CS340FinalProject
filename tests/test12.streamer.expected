Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: file1data
            Type: text
            File Read into variable: file1data
                List Access
                    Identifier: args
                    Number: 0.0
        Declaration statement: file2data
            Type: text
            File Read into variable: file2data
                List Access
                    Identifier: args
                    Number: 1.0
        If Statement
            Binary operator '=='
                Identifier: file1data
                Identifier: file2data
            Statement List
                Print Statement
                    Quoted string: "A match!"
            Statement List
                Print Statement
                    Quoted string: "Different!"
